client/my-sites/theme/main.jsx

```diff
@@ -173,6 +178,11 @@ class ThemeSheet extends Component {
 		if ( this.props.themeId !== prevProps.themeId ) {
 			this.scrollToTop();
 		}
+
+		if ( this.state.disabledButton !== this.isLoading() ) {
+			// eslint-disable-next-line react/no-did-update-set-state
+			this.setState( { disabledButton: this.isLoading() } );
+		}
```

Comment:
This "props to state sync" code should also be in `componentDidMount`. If `this.isLoading()` is `false` on initial mount, and then the component never rerenders again, then `state.disabledButton` would stay `true` and we don't want that. The "never rerenders again" case is theoretical, the component will almost certainly be rerendered, but it's good to make components resilient.

Reply:
Added at https://github.com/Automattic/wp-calypso/pull/77696/commits/bce2f0d71f8be5defe3f63328f2b2ea0e5c18418